### ISSUE TITLE
Bullet list in text on slate blue background isn't automatically white

### ISSUE THREAD

**Issue by meols** [equinor/energyvision#1641]
The "normal" text in a text component with slate blue background on a topic page is automatically white, while the bullet list isn't. Bullet list should behave the same way.
![image.png](https://images.zenhubusercontent.com/5fbbf4f2bab8ae744cb6560f/ff194429-5924-4b92-ba53-4e26be4f0f6e)

**Comment by padms** [comment]
LGTM

**Comment by meols** [comment]
Great! Thanks for fixing @millianapia :smile:
